ee/clickhouse/models/cohort.py

```diff
@@ -276,7 +281,11 @@ def recalculate_cohortpeople(cohort: Cohort):
         size_before=before_count[0][0],
     )
 
-    cohort_filter = GET_PERSON_IDS_BY_FILTER.format(distinct_query="AND " + cohort_filter, query="")
+    cohort_filter = GET_PERSON_IDS_BY_FILTER.format(
+        distinct_query="AND " + cohort_filter,
+        query="",
+        GET_TEAM_PERSON_DISTINCT_IDS=get_team_distinct_ids_query(cohort.team_id),
```

Comment:
Does this work as you'd expect?

The format happening here: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/sql/person.py#L213 seems to supercede what you're doing here^^

Reply:
It should - I updated this at https://github.com/PostHog/posthog/pull/7577/files#diff-8c1a8fdf464c1e77c60c9d3a113fa05e04ba342aa680324f9d037e936f3dfd6cR222

Reply:
nvm, missed the update there